src/components/FunctionalComponent.js

```diff
@@ -0,0 +1,11 @@
+const FunctionalComponent = () => {
+  return (
```

Comment:
In function components the word "return" can be omitted
Suggested change
```suggestion
import React from "react";

const FunctionalComponent = () => (
  <>
    <h1>Hello World!</h1>
    <span>It's a Functional.Component</span>
    <hr />
  </>
);

export default FunctionalComponent;
```